worlds/ladx/__init__.py

```diff
@@ -66,6 +68,15 @@ class LinksAwakeningWebWorld(WebWorld):
     )]
     theme = "dirt"
     option_groups = ladx_option_groups
+    options_presets = {
+        "Keysanity": {

```

Comment:
I would probably change this, maps, compasses, and stone beaks are not very "key" related. You could add in Instruments and call it "All Shuffle" or something maybe?

Reply:
Well, I think upstream's preset is poorly named, then. "Shuffle Dungeon Items" maybe?

Reply:
I agree with the points made by Medic and Scipio, HOWEVER: At the moment, presets always change every option. As a result, I think it's fine to add other options that seem to have a similar "spirit"